Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "gateway-server"
```

Comment:
why not `gateway-proto` as the repo is named?

Reply:
Ah, I've copied it from my playground and forgot to change. Fixed